packages/forms-web-app/src/controllers/lpa-dashboard/your-appeals.js

```diff
@@ -13,14 +13,19 @@ const {
 } = require('../../lib/views');
 const { baseHASUrl } = require('../../dynamic-forms/has-questionnaire/journey');
 
-const { getAppealsCaseData } = require('../../lib/appeals-api-wrapper');
+const {
+	getAppealsCaseDataV2,
+	getDecidedAppealsCaseDataV2
+} = require('../../lib/appeals-api-wrapper');
 
 const getYourAppeals = async (req, res) => {
 	let appealsCaseData = [];
 
 	const user = getLPAUserFromSession(req);
 
-	appealsCaseData = await getAppealsCaseData(user.lpaCode);
+	appealsCaseData = await getAppealsCaseDataV2(user.lpaCode);
+
+	const decidedAppeals = await getDecidedAppealsCaseDataV2(user.lpaCode);
```

Comment:
There's a separate API for count if that is all we need at this point.

`/api/v2/appeal-cases/count`

Reply:
Ah I thought there was something like that, meant to switch previously, have switched now.